src/SFA.DAS.Roatp.CourseManagement.Web/Validators/AddAStandard/TrainingLocationListViewModelValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using SFA.DAS.Roatp.CourseManagement.Web.Models.AddAStandard;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Validators.AddAStandard
+{
+    public class TrainingLocationListViewModelValidator : AbstractValidator<TrainingLocationListViewModel>
+    {
+        public const string TrainingLocationErrorMessage = "You must add a training location";
+        public TrainingLocationListViewModelValidator()
+        {
+            RuleFor(p => p.FirstLocation)
```

Comment:
Why not check if the list is empty or not, instead of first location's name? If you change this, remove first location from model

Reply:
As it's a post event, the list is always empty.  I tried several alternatives.  I will leave this till last and if I can't figure it out, I'll do a call with you

Reply:
This is now fixed as requested